### ISSUE TITLE
[QUESTION]: multi-class classification

### ISSUE THREAD

**Issue by llzhu** [chemprop/chemprop#391]
Hi,
I went through some examples with dataset_type'=classification; all of them are binary classification (target=0,1).
Does ChemProp support multi-class classification? if it does, how is it implemented?
Thanks.  -Leiming

**Comment by kevingreenman** [comment]
Yes, the multiclass option is described in the [data](https://github.com/chemprop/chemprop#data) section of the README. Copying the description here:

```
Targets are integers (starting with zero) indicating which class the datapoint belongs to, out of a total number of exclusive classes indicated with --multiclass_num_classes <int>.
```

For example, to train a model on data with 3 exclusive classes, you would use the arguments `--dataset_type multiclass --multiclass_num_classes 3`.

As described in the [loss functions](https://github.com/chemprop/chemprop#loss-functions) section of the README, multiclass is trained using the cross entropy loss as a default, but it also supports training with a soft version of Matthews Correlation Coefficient (MCC).
